db/models/Memory.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+// working on data structure:
```

Comment:
I love your comments! Makes it so much easier to get through your code 👍

Reply:
Thanks, I saw it first in your code and it made sense to adapt moving forward. So kudos back to you!